### PR TITLE
Milestone 1: Create Model ERD 

### PR DESCRIPTION
**Pull request by Ceejayski**
In this milestone we created the Entity Relationship Diagram for the following models below:
- User model
- Friendship model
- Post Model
- Like Model

### CODE REVIEW

**Review by adaorachi — CHANGES_REQUESTED** [review]
### Hi @Ceejayski and @enaburekhan :wave: 
You've done quite a good job on this milestone. Well done! :+1: 
However, there are a couple of changes to implement in order to get this project approved.

---

### CHANGES REQUIRED :recycle: 

---

- [ ] Github action flagged an error in the stylesheet of this project as you can see above. Kindly ensure the error is fixed.
- [ ] The relationship between the user model and the friendship model should be a one-to-many relationship. A user can have many followers and a user can also have many followees. With this, kindly adjust the arrow connection signs to reflect this. 
And since you have indicated two foreign keys in the friendship model, there should also be two kinds of relationships between the user and each of the foreign keys in the friendship model. Kindly ensure this is reflected as well.

### After implementing the changes, you can go ahead and request another code review.

Should you have any questions, kindly reach out to me on Slack with the username @adaorachi

Happy Coding :relaxed: 

MaryAnn Chukwuka

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I like your ERD. Almost everything looks good after implementing the required changes by the last reviewer. However, there's one more fix to make before this project can be approved. Let's dive in.__

**[ REQUIRED CHANGES ]**

- [ ] Kindly rename the `table_id` for each table as `ID`. For instance, on your `User` table, the id of the table (which you named user_id) should be renamed to `ID`. This makes more sense than repeating the prefix `user_` for the ID column in the user table. 
- This is applicable to all other tables. i.e.
  - On the friendship table, the `friendship_id` column (which is the primary key) should be named just `ID` on the friendship table, etc.
- Note that this only affects the primary keys and not the foreign keys.

- [ ] There are many to many relationships between the user and friendship. This makes sense as we can easily say that:
  - A user can send zero or more friend requests, a friend request belongs to one and only one user.
  - A user can also receive zero or more friend requests, whereas, an incoming friend request belongs to one and only one user.
Therefore, adjust the second relationship between user and friendship, such that the `||` points to the user table, and the `o<-` points to the friendship table.

**Check the diagram below for more explanation**





* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by acushlakoncept — APPROVED** [review]
# Status: PROJECT APPROVED  🎉 🚀

### Great job guys!!! 👏  
Congratulations on completing the first milestone of this project.

I will go ahead and mark this as complete, feel free to merge when you are ready. All the best with the next milestone.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Pull request merged**
